app/Console/Kernel.php

```diff
@@ -33,6 +33,9 @@ protected function schedule(Schedule $schedule)
         $schedule->command('backup:monitor')->daily()->at('04:00')->when(function() {
             return config('app.enable_backup_monitor');
         });
+        $schedule->command('orcid:update')->weekly()->at('05:00')->when(function() {
```

Comment:
@betsyecastro I think it would be good to add an explicit `sundays()` declaration to the chain here, so that we can see exactly what day when looking at the code without having to look up in the docs. e.g. `->weekly()->sundays()->at('05:00')`

Reply:
@wunc The requested change has been pushed to this branch in the last commit.